src/content/configuration/configuration-types.md

```diff
@@ -44,7 +45,7 @@ One option is to export a function from your webpack config instead of exporting
 
 ## Exporting a Promise
 
-webpack will run the function exported by the configuration file and wait for a Promise to be returned. Handy when you need to asynchronously load configuration variables.
+webpack will run the function exported by the configuration file and wait for a Promise to be returned. If you have an array of Promises you can wrap it in `Promises.all([])`. Handy when you need to asynchronously load configuration variables.
```

Comment:
There is a typo here, it is `Promise` not `Promises`. besides this reads better as:
```suggestion
webpack will run the function exported by the configuration file and wait for a Promise to be returned. Handy when you need to asynchronously load configuration variables.
```